stable/prometheus-statsd-exporter/values.yaml

```diff
@@ -0,0 +1,43 @@
+# Default values for prometheus-statsd-exporter.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: prom/statsd-exporter
+  tag: v0.7.0
+  pullPolicy: IfNotPresent
+
+service:
+  type: ClusterIP
+
+webPort: 9102
+udpPort: 9125
+tcpPort: 9125
+
+podAnnotations: {}
+
+priorityClassName: ""
+
+resources: {}
+
+nodeSelector: {}
+
+affinity: {}
+
+tolerations: []
+
+statsdMappingConfig: |-
+  defaults:
+    timer_type: histogram
+    buckets: [.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10]
+    quantiles:
```

Comment:
why? with `timer_type: histogram` this will be unused. Anyone overriding the default needs to override the whole config anyway

Reply:
True, not relevant for `histogram`, but meant as a default for when the `timer_type` is set to `summary`. However, since these are the defaults anyway (therefore redundant), and the default `timer_type` is explicitly set to be `histogram`, I will remove the default quantiles to avoid any confusion.

The only truly non-default value here is `timer_type: histogram`. If you think everything else should be removed, let me know. No point in adding redundant details.